prime-router/docs/api/waters-delivery.yml

```diff
@@ -18,7 +18,7 @@ servers:
 paths:
   /waters/{organization}/deliveries:
     get:
-      operationId: getAvailableReports
+      operationId: getDeliveries
```

Comment:
getdeliveries

Reply:
i put it like that to match the method in code